### ISSUE TITLE
can not preview when no folder is opened

### ISSUE THREAD

**Issue by bobandyou** [Enter-tainer/typst-preview-vscode#60]
Type: <b>Bug</b>

It cann't work, no preview but white blank.

Extension version: 0.6.0
VS Code version: Code 1.80.0 (660393deaaa6d1996740ff4880f1bad43768c814, 2023-07-04T15:06:02.407Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E5-1620 0 @ 3.60GHz (8 x 3591)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.93GB (20.76GB free)|
|Process Argv|--crash-reporter-id 16746ae1-398d-4caa-8977-522b8b5cf4f3|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30610679
282f8724:30602487
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
2i9eh265:30646982
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
57b77579:30736110
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
showindicator:30785052
pythongtdpath:30769146
bgfeh915:30780428
pythonnosm12tcf:30779713
pythonidxpt:30784022
pythonnocebcf:30776496
e537b577:30786199

```

</details>

**Comment by Enter-tainer** [comment]
can you provide your typst source file?

**Comment by Enter-tainer** [comment]
I would close this issue because no useful information is provided. Feel free to reopen it if you can provide more info.

**Comment by yuioto** [comment]
I may have reproduced this issue. I found that it doesn't work when a folder is not opened, and the screen is blank.

I think this may be because a cache file cannot be created without opening a folder.

## reproduced
1. Run `code test.typ` on shell.
2. Open preview.
3. preview is blank.

## system info
I use windows 10 22H2 19045.2788
vscode 1.80.1 user setup

I'm sorry for my poor English.

**Comment by Enter-tainer** [comment]
It has been fixed in v0.6.2. It will be available on vscode marketplace hopefully within 30mins.
